<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>feedIO</class>
 <widget class="QMainWindow" name="feedIO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>feedIO</string>
  </property>
  <property name="windowIcon">
   <iconset resource="feedIOicons.qrc">
    <normaloff>:/images/feedIO.png</normaloff>:/images/feedIO.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Feed</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboFeed">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>250</width>
              <height>36</height>
             </size>
            </property>
            <property name="maxVisibleItems">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Interest</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboTopic">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>250</width>
              <height>36</height>
             </size>
            </property>
            <property name="maxVisibleItems">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCommandLinkButton" name="btnUp">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>36</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;UP</string>
          </property>
          <property name="icon">
           <iconset resource="feedIOicons.qrc">
            <normaloff>:/images/upVote.png</normaloff>:/images/upVote.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="btnDown">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>36</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;DOWN</string>
          </property>
          <property name="icon">
           <iconset resource="feedIOicons.qrc">
            <normaloff>:/images/downVote.png</normaloff>:/images/downVote.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeWidget" name="listUnread">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>300</width>
          <height>500</height>
         </size>
        </property>
        <property name="indentation">
         <number>-10</number>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Important &amp; Unread</string>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
       </widget>
       <widget class="QTreeWidget" name="listOld">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>300</width>
          <height>200</height>
         </size>
        </property>
        <property name="indentation">
         <number>-10</number>
        </property>
        <column>
         <property name="text">
          <string>Everything else</string>
         </property>
         <property name="font">
          <font>
           <family>Sans Serif</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWebView" name="viewArticle" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="url" stdset="0">
        <url>
         <string>http://feedio.org</string>
        </url>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Feeds</string>
    </property>
    <addaction name="actionAddFeed"/>
    <addaction name="actionRemoveFeed"/>
    <addaction name="actionFetchFeed"/>
    <addaction name="actionFetchAllFeeds"/>
    <addaction name="actionManageFeeds"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuArticle">
    <property name="title">
     <string>&amp;Article</string>
    </property>
    <addaction name="actionUpVote"/>
    <addaction name="actionDownVote"/>
    <addaction name="actionRead"/>
    <addaction name="actionVisitPage"/>
    <addaction name="actionUnread"/>
   </widget>
   <widget class="QMenu" name="menuFeedIO">
    <property name="title">
     <string>feedI&amp;O</string>
    </property>
    <addaction name="actionMinimizeToTray"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTopics">
    <property name="title">
     <string>&amp;Interests</string>
    </property>
    <addaction name="actionAddTopic"/>
    <addaction name="actionRemoveTopic"/>
    <addaction name="actionManageTopics"/>
   </widget>
   <addaction name="menuFeedIO"/>
   <addaction name="menuFile"/>
   <addaction name="menuTopics"/>
   <addaction name="menuArticle"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <italic>false</italic>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddFeed"/>
   <addaction name="actionRemoveFeed"/>
   <addaction name="separator"/>
   <addaction name="actionFetchFeed"/>
   <addaction name="actionFetchAllFeeds"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
   <addaction name="actionVisitPage"/>
   <addaction name="actionRead"/>
   <addaction name="separator"/>
   <addaction name="actionUpVote"/>
   <addaction name="actionDownVote"/>
  </widget>
  <action name="actionAddFeed">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/addFeed.png</normaloff>:/images/addFeed.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Feed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRemoveFeed">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/removeFeed.png</normaloff>:/images/removeFeed.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove Feed</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>FeedIO Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/feedIO.png</normaloff>:/images/feedIO.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About feedIO</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionUpVote">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/upVote.png</normaloff>:/images/upVote.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Up vote </string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDownVote">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/downVote.png</normaloff>:/images/downVote.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Down Vote</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRead">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/read.png</normaloff>:/images/read.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Read</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFetchFeed">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/fetchFeed.png</normaloff>:/images/fetchFeed.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fetch Feed</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFetchAllFeeds">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/fetchAll.png</normaloff>:/images/fetchAll.png</iconset>
   </property>
   <property name="text">
    <string>Fetch &amp;All Feeds</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionVisitPage">
   <property name="icon">
    <iconset resource="feedIOicons.qrc">
     <normaloff>:/images/visitPage.png</normaloff>:/images/visitPage.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Visit Page</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionManageFeeds">
   <property name="text">
    <string>&amp;Manage Feeds</string>
   </property>
  </action>
  <action name="actionManageTopics">
   <property name="text">
    <string>&amp;Manage Topics</string>
   </property>
  </action>
  <action name="actionAddTopic">
   <property name="text">
    <string>&amp;Add Topic</string>
   </property>
  </action>
  <action name="actionRemoveTopic">
   <property name="text">
    <string>&amp;Remove Topic</string>
   </property>
  </action>
  <action name="actionMinimizeToTray">
   <property name="text">
    <string>Mi&amp;nimize to tray</string>
   </property>
  </action>
  <action name="actionUnread">
   <property name="text">
    <string>&amp;Mark as Unread</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="feedIOicons.qrc"/>
 </resources>
 <connections/>
</ui>
